.github/workflows/review.yml

```diff
@@ -23,13 +23,11 @@ jobs:
           if git diff --name-only origin/${{ github.base_ref }} HEAD | grep -Eq '*\.js|node\.d\.plugin\.in' ; then
             echo 'run_eslint=1' >> $GITHUB_ENV
           fi
-      - name: Run eslint
+      - name: Install eslint
+        run: npm install eslint -D
+      - name: Run ESLint
         if: env.run_eslint == 1
-        uses: reviewdog/action-eslint@v1
-        with:
-          github_token: ${{ secrets.GITHUB_TOKEN }}
-          reporter: github-pr-check
-          eslint_flags: '.'
+        run: ./node_modules/.bin/eslint . --ext .js,.jsx,.ts,.tsx
```

Comment:
If possible I’d like to keep using the ReviewDog actions, as they provide proper annotations on the diff itself for any warnings. Simply ensuring ESLint is installed and properly updating the flags we are passing in _should_ be enough to make it work correctly.

Reply:
I decided after all, not to mix the eslint changes needed with this PR, as the purpose of this addition is to add a simple UI test and an action running it. Regarding linting, we will treat it in a subsequent PR, after discussing it with the FE team.